app/controllers/application_controller.rb

```diff
@@ -292,8 +292,9 @@ def run_scheduler
   end
 
   def update_persistent_announcements
-    @persistent_announcements = Announcement.where(persistent: true)
-                                            .where("course_id = ? OR system = ?", @course.id, true)
+    @persistent_announcements = Announcement
+      .where(persistent: true, course_id: @course.id)
+      .or(Announcement.where(persistent: true, system: true))
```

Comment:
random comment: can you move `Announcement.where(course_id: ...).or(Announcement.where(system: true))` into a scope and use that instead? Though I'd imagine it would need to be the start of the chain

Reply:
Hey Nic, what a surprise!! :D It's definitely been a while hahah. 

Yeah, I agree that using a scope will help with the code repetition here, haven't used that in a long time so did not think about that one. Great suggestion!!

Reply:
Happened to chance across this, let's catch up separately haha